### DISCUSSION TITLE
Error ketika kirim file csv di leaderboard

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#639] [Q&A] [answered]
setelah diinput file csv di leaderbord dihasilkan error sbb ;

<img width="379" alt="image" src="https://user-images.githubusercontent.com/123436963/232326006-74f76c24-2d1c-435d-9603-b174537c0e25.png">

saya sdh mengikuti tipe data seperti file submission-example yaitu bertipe character untuk ketiga kolomnya ;

<img width="434" alt="image" src="https://user-images.githubusercontent.com/123436963/232326222-24699fa5-9866-4ee7-9615-2d1f6c2c1a0a.png">

serta penulisan nama filenya yaitu  submission-scot

Bagaimana cara memperbaikinya ?

Terimakasih

**Comment by amaliasf** [comment, marked as answer]
Dear Pak @Micoalgo 

Terimakasih atas pertanyaannya. Berdasarkan screenshot yang Bapak, kemungkinan besar dikarenakan masih ada index yang ikut tersimpan dalam file .csv. Pastikan ketika menyimpan file Bapak menambahkan paramateter `row.names = F`.

Berikut contoh code penyimpanan object menjadi file .csv:

`write.csv(submission, "submission-david.csv", row.names = F)`

Boleh silahkan dicoba terlebih dahulu, dan diinfokan kembali kalau masih terkendala. Semoga berhasil Pak untuk submissionnya.

**Reply by Micoalgo** [reply]
Terimakasih jawabannya
